deploy/local/vars_lnx.sh

```diff
@@ -223,6 +224,19 @@ if [ ! -d $__daal_tmp_dir ]; then
     __daal_tmp_dir=${component_root}
 fi
 
+
+PLATFORM=$(bash dev/make/identify_os.sh)
```

Comment:
this will not work. vars scripts are executed for configured build already.  i.e. one that get packages and provided to users - there is no dev/make/identify_os.sh script that will be available there. 

i think arch identification have to be implemented here instead